reds/debug.reds

```diff
@@ -0,0 +1,11 @@
+@addMethod(PlayerPuppet)
+public func TryTDBID() -> Void {
```

Comment:
I'd drop the `@addMethod(PlayerPuppet)` to make it simpler since it's just a dummy demo
and maybe rename it to `DemoFunc` or something

Reply:
resolved across [f152c5e](https://github.com/jac3km4/red4ext-rs-example/pull/1/commits/f152c5ef039a4d56b288917d78a1766f7f1be61b)..[5196fee](https://github.com/jac3km4/red4ext-rs-example/pull/1/commits/5196fee9bbbf68c8ec70c02cdc5cb2e7cb6c977a)